FilePersistence/src/version_control/merge/ChangeGraph.cpp

```diff
@@ -448,14 +446,105 @@ void ChangeGraph::removeLabelConflictsBetweenChildren(Model::NodeIdType parentId
 	}
 }
 
-void ChangeGraph::updateIncomingLabels(Model::NodeIdType /*parentId*/, IdToLabelMap /*labelMap*/)
+void ChangeGraph::updateListChanges(Model::NodeIdType parentId, IdToLabelMap labelMap, GenericTree* baseTree)
```

Comment:
I don't like the word `list` in the name of this method. While we use it for lists, it could technically be used for anything else that has unique labels.

Let's call this `updateLabelsOfChangesTo`